vizro-core/src/vizro/static/js/models/dashboard.js

```diff
@@ -3,7 +3,6 @@ function update_dashboard_theme(theme_selector_checked) {
     "data-bs-theme",
```

Comment:
FYI @petar-qb this is an example of a callback that we always want to run even when it's not explicitly triggered. Otherwise we end up with no `data-bs-theme` specified. Currently this works well but it wouldn't if this callback was not triggered automatically unless we set `:root` to be dark theme as well (which might happen).

Reply:
> The `Output("dashboard-container", "className")` in this file shouldn't be there any more.

@antonymilne @petar-qb  - If I remove the Output as suggested above, this callback doesn't get triggered anymore and we end up with no `data-bs-theme` specified. The layout doesn't match anymore because of that. 

I assume if we would set` :root` to be the dark theme, this issue doesn't appear anymore, but if we stick with the light theme - how can I always trigger this callback if we simultaneously want to remove the `Output("dashboard-container", "className")`? I guess I would have to add some placeholder Output there in which case I would just leave the current Output in?

Reply:
This is strange, and probably a bug from the dash side. However, I suggest to leave the Output then (or to target some other dummy component, but not mandatory).

Reply:
Oh that's weird. I guess it's either to do with that complicated Dash scheme that I never understand about which callbacks get triggered depending on whether the output is already on the page or just a bug like @petar-qb said.

Let's put some current `Output` there with a comment that it doesn't do anything but is needed to run the callback and put `return window.dash_clientside.no_update` in the callback.

Setting `:root` here won't help us regardless of what colour it is because if you've specified `dashboard.theme = <not the default :root theme>` then we need the `update_dashboard_theme` to run to set it correctly. So this callback has to always run whatever we do.